clients/lite-xl/tests/init_completion_resolve_test.lua

```
@@ -0,0 +1,880 @@
+local function play_response(server, method, response)
+  for index, entry in ipairs(server.outbound) do
+    if entry.kind == "request" and entry.method == method then
+      table.remove(server.outbound, index)
+      if entry.callback then entry.callback(server, response or {}) end
+      return true
+    end
+  end
+  return false
+end
+
+local INCREMENTAL_CAPS = {
```

Luacheck (1.2.0)

[warning] unused variable 'INCREMENTAL_CAPS'

(W211)